org.mixedrealitytoolkit.uxcore/SeeItSayIt/SeeItSayItLabelEnabler.cs

```diff
@@ -38,6 +51,32 @@ public GameObject SeeItSayItLabel
             set => seeItSayItLabel = value;
         }
 
+#if UNITY_LOCALIZATION_PRESENT
+        [SerializeField]
+        [Tooltip("The LocalizedString that define the label pattern. Use a smart string with one argument that will be replaced by the button's speech recognition keyword (e.g: \"Say '{0}'\").")]
+        private LocalizedString localizedPattern;
+#else
+        [SerializeField]
+        [Tooltip("The patern for the see-it say-it label using string.Format()")]
+        private string pattern = "Say '{0}'";
```

Comment:
Do we have any prefabs with this script on them in an MRTK package? If so, I wonder if this will lead to Unity logs (`Saving Prefab to immutable folder is not allowed`) around trying to reserialize immutable files.

Reply:
Great question @keveleigh! I missed that one.

Reply:
Yes, some prefab use this script ("Action Button", "Action Button Checkbox", "SimpleActionButton" and maybe more).

I use this improvement in my main project for a long time, and never seen this message. In what situation to you think it can happen? User can't save (= reserialize) prefabs that are in a package.